tests/test_mathematical.py

```diff
@@ -4,6 +4,8 @@
 
 import numpy
 
+import unittest
```

Comment:
We run tests by `pytest`. So we are able to not use `unittests` like in `cupy` tests.